guides/plugins/plugins/administration/add-shortcuts.md

```diff
@@ -0,0 +1,55 @@
+## Adding Shortcuts
+
+Shortcuts in the Shopware 6 are defined on a Component basis. This guide will show you how.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance and full access to both the files.
+Preferably a registered module and custom component.
```

Comment:
Is this a continuation of the previous statement ? 
If so, seperate them by comma or add an **and**